functions/system.sh

```diff
@@ -217,6 +220,19 @@ pine64_platform_scripts() {
   fi
 }
 
+pine64_gpio_permissions() {
+  cond_redirect groupadd gpio
+  cond_redirect cp $BASEDIR/includes/PINE64-80-gpio-noroot.rules /etc/udev/rules.d/80-gpio-noroot.rules
+  cond_redirect sed -i -e '$i \chown -R root:gpio /sys/class/gpio \n' /etc/rc.local
+  cond_redirect sed -i -e '$i \chmod -R ug+rw /sys/class/gpio \n' /etc/rc.local
+} 
+
+pine64_fix_system_binding() { # This will maybe be fixed upstreams some day. Keep an eye open.
+  cond_redirect apt-get install -y udev:armhf
+  cond_redirect ln -s /lib/arm-linux-gnueabihf/ /lib/linux-arm
+  cond_redirect ln -s /lib/linux-arm/libudev.so.1 /lib/linux-arm/libudev.so
+}
```

Comment:
Please a logging line, the usual "timstamp, openHABian, Doing this and that... " + "OK" would be fine.
Also please add some error checking and testing. Those `ln` calls will e.g. fail if the files already exist. I'm looking at future additions in the base system here...

Please replace apt-get by apt. Not that it makes a difference but all other functions use apt and it's a matter of consistency.